<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditCalc</class>
 <widget class="QDialog" name="CreditCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Credit Calc</string>
  </property>
  <widget class="QPushButton" name="calculate">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>490</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Расчёт</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_calculation_result">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Результат расчёта</string>
   </property>
  </widget>
  <widget class="QListWidget" name="list_values">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>451</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>271</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_term">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_total_amount">
      <property name="text">
       <string>Сумма, руб</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_term">
      <property name="text">
       <string>Срок, мес.</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_amount"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_rate"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_interest_rate">
      <property name="text">
       <string>Ставка, %</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="type">
      <property name="text">
       <string>Тип</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_annuity">
      <property name="text">
       <string>аннуитетный</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_differentiated">
      <property name="text">
       <string>дифференцированный</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>70</y>
     <width>401</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_total_payment">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_overpayment"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_overpayment">
      <property name="text">
       <string>Переплата, руб</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_total_payment">
      <property name="text">
       <string>Общая выплата, руб</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
